ui/components/multichain/network-manager/index.scss

```diff
@@ -38,6 +39,17 @@
   }
 }
 
+.network-manager__all-popular-networks {
+  .mm-avatar-base {
+    background-color: transparent;
+    border-color: transparent;
+
+    .mm-icon {
+      --icon-size: 30px;
+    }
+  }
+}
```

Comment:
The globe icon in the NetworkManager uses the an existing component that uses Avatars. This tweaks the Avatar so that the icon does not render a background and also ensures height match the other avatar heights (in figma)
<img width="500" height="204" alt="Screenshot 2025-08-22 at 20 53 58" src="https://github.com/user-attachments/assets/23a52409-987d-4b00-aa6f-b51b5110818f" />